frontend_app_thread/src/container/Thread.jsx

```diff
@@ -374,6 +377,47 @@ export class Thread extends React.Component {
     props.appContentRestoreDelete(state.content, this.setState.bind(this), state.config.slug)
   }
 
+  handleClickEditComment = (comment) => {
```

Comment:
Couldn't this code be factorized in `appContentFactory`?

Reply:
`appContentEditComment` is an `appContentFactory` function